Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/cell/PostCell.java

```diff
@@ -436,7 +436,8 @@ private void bindPost(Theme theme, Post post) {
         if (ChanSettings.shiftPostFormat.get()) {
             comment.setVisibility(isEmpty(commentText) ? GONE : VISIBLE);
         } else {
-            comment.setVisibility(isEmpty(commentText) && post.images == null ? GONE : VISIBLE);
```

Comment:
I have no idea what the hell is going on here. The logic got completely fucked and this is completely wrong now.

Reply:
The `isEmpty(commentText) && post.images == null` is always false, according to lint. The `post.images == null` condition was always false when reached. I'll revert it back anyway, to be safe.

Reply:
The lint is only using the initial set of `post.images = null` to determine the truth value of the statement, which is only partially correct; if post.images is modified later then this statement may return true (which it does).